Gemfile.lock

```diff
@@ -78,29 +78,29 @@ GEM
       public_suffix (>= 2.0.2, < 5.0)
     airbrussh (1.4.0)
       sshkit (>= 1.6.1, != 1.7.0)
-    annotate (3.0.3)
+    annotate (3.1.1)
       activerecord (>= 3.2, < 7.0)
       rake (>= 10.4, < 14.0)
-    apipie-rails (0.5.16)
+    apipie-rails (0.5.17)
       rails (>= 4.1)
     arbre (1.2.1)
       activesupport (>= 3.0.0)
     ast (2.4.0)
     aws-eventstream (1.0.3)
-    aws-partitions (1.258.0)
-    aws-sdk-core (3.86.0)
+    aws-partitions (1.289.0)
+    aws-sdk-core (3.92.0)
       aws-eventstream (~> 1.0, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.27.0)
+    aws-sdk-kms (1.30.0)
```

Comment:
### [aws-sdk-kms](https://github.com/aws/aws-sdk-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.27.0 → 1.30.0

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-kms/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-kms)_]





<details>
<summary>Commits</summary>

A change of **43** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/1.27.0...1.30.0).

These are the first 10 commits:
- (577ac54) [adds public IP option to instances in VPC](https://github.com/aws/aws-sdk-ruby/commit/577ac54d408ba75f8616dd27decac4eeabf59ced)
- (8c4a152) [Merge branch 'master' of https://github.com/aws/aws-sdk-ruby](https://github.com/aws/aws-sdk-ruby/commit/8c4a1521e582c1f27fbd975ef2240517009d0e6f)
- (844573b) [delete the associate_public_ip_address option from create in…](https://github.com/aws/aws-sdk-ruby/commit/844573bac0b87a1bf43819dae833e784f1af5d2a)
- (4d55bdb) [Updated API definition of Monitoring field in request_spot_i…](https://github.com/aws/aws-sdk-ruby/commit/4d55bdb026b9dca54eee75e02233c1120af64bb8)
- (4a5beef) [Allow the ability to supply private IP to the associate meth…](https://github.com/aws/aws-sdk-ruby/commit/4a5beef88a05ed291a7a862ef3f3977b836e60e4)
- (f6fce2f) [Support RawMessageDelivery on SNS subscriptions.](https://github.com/aws/aws-sdk-ruby/commit/f6fce2f7d6989b980a9eda77b95ba3fa2414f17b)
- (9055bdc) [Now using File.size to determine file size for File/Tempfile…](https://github.com/aws/aws-sdk-ruby/commit/9055bdc0057b00e823695b35266d1b54466b0cb4)
- (d0a8bec) [Updating AWS::CloudWatch to use sigv4.](https://github.com/aws/aws-sdk-ruby/commit/d0a8becbad60290faa6c26c5d4194df90e069807)
- (b78e18f) [Added a nokogiri dependendency to the Gemfile for local deve…](https://github.com/aws/aws-sdk-ruby/commit/b78e18f7efe30f30c74cafdba0bf282d31e3a49d)
- (5c5758f) [Fixed a bug with Ruby 1.8 and how StringIO#path.](https://github.com/aws/aws-sdk-ruby/commit/5c5758fa0499f47d1582ce974889c83d2c0f64dd)


</details>